src/include/daos/object.h

```diff
@@ -51,9 +51,14 @@ struct daos_obj_shard_md {
 /**
  * object layout information.
  **/
+struct daos_target_id {
+	uint32_t	ti_tgt;
```

Comment:
may comment similar with "tgt id unique in pool (co_id), not the tgt_idx (co_index) in ti_rank".

And are you sure you need the co_id rather than co_index? or not both? may comment about the usage in commit msg.

Reply:
Hmm, maybe it should output both id and index here. I will update the patch.